packages/graphql/src/index.ts

```diff
@@ -0,0 +1,41 @@
+import { createHttpApp, Scenarios, Service, Simulator } from '@simulacrum/server';
+import { json } from 'express';
+import { GraphQLSchema } from 'graphql';
+
+import { playground, graphql as graphQLHandler } from './handlers';
+import { ContextCreator, DynamicImport, GraphQLOptions, Options } from './types';
+
+function dynamicImport<TExport>(options: DynamicImport) {
+  // eslint-disable-next-line @typescript-eslint/no-var-requires
+  let result = require(options.module)[options.export] as TExport;
+
+  if (!result) {
+    console.warn(`export not found: "${JSON.stringify(options)}"`);
+  }
```

Comment:
Wasn't entirely sure of the error handling strategy for these simulators; this should probably be improved?